DDL.sql

```diff
@@ -0,0 +1,263 @@
+
+CREATE TABLE BMSNumThermAttemptsMade(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP NOT NULL
+);
+
+CREATE TABLE BMSLastAttemptTime(
+    ID SERIAL PRIMARY KEY,
+    time INTEGER NOT NULL,
+    timeStamp TIMESTAMP NOT NULL,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSThermAttemptTime(
+    ID SERIAL PRIMARY KEY,
+    time INTEGER NOT NULL,
+    timeStamp TIMESTAMP NOT NULL,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSTotalVoltage(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSBatteryVoltage(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSCurrent(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSThermistorTemp(
+
+    ID SERIAL,
+    thermNumber INTEGER,
+    PRIMARY KEY (ID, thermNumber),
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSMinPackTemp(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER,
+    packId INTEGER,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSMaxPackTemp(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER,
+    packId INTEGER,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSBqInternalTemp(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSBqTemp(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    bqId INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSCellVoltage(
+    ID SERIAL,
+    cellId INTEGER,
+    PRIMARY KEY (ID, cellId),
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSMinCellVoltage(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    cellId INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSMaxCellVoltage(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    cellId INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSBqStatus(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSErrorRegister(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSLastCheckedThermNum(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE IMUVector(
```

Comment:
Can you explain what this even means? How are you representing each of these vectors as an integer?